pkgs/servers/monitoring/prometheus/default.nix

```diff
@@ -59,7 +59,7 @@ in buildGoPackage rec {
     cp -a $src/console_libraries $src/consoles $out/etc/prometheus
   '';
 
-  doCheck = true;
+  doCheck = !stdenv.isDarwin; # https://hydra.nixos.org/build/130673870/nixlog/1
```

Comment:
Seems likely since upstream is also not testing on macOS: https://app.circleci.com/pipelines/github/prometheus/prometheus/9912/workflows/47d71aaf-7e51-4dc6-9ea7-0f8611f75e61/jobs/43784